source/scoped-mutex-traits/Cargo.toml

```diff
@@ -1,4 +1,22 @@
 [package]
 name = "scoped-mutex-traits"
-version = "0.1.0"
+version = "1.0.0"
+authors = [
+    "James Munns <james@onevariable.com>",
+    "Eliza Weisman <eliza@elizas.website>",
+]
 edition = "2021"
+readme = "../../README.md"
+repository = "https://github.com/tosc-rs/scoped-mutex"
+description = "An abstraction over closure-based mutexes"
+license = "MIT OR Apache-2.0"
+categories = [
+    "embedded",
+    "no-std",
+]
```

Comment:
nit, take it or leave it: IIRC you can define all of this metadata at the workspace-level and just use `authors.workspace = true`, which might make it a little easier to change. i don't actually care if we use that or not, though.